webnn/resources/test_data/constant.json

```diff
@@ -616,7 +616,7 @@
       "name": "constant float32 4D tensor of int8 type",
       "inputs": {
         "start": {
-          "data": 0.22992068529129028,
+          "data": -5.520709991455078,
           "type": "float32"
```

Comment:
Wait, this isn't supported, mixing and matching different data types like float32 and int8. If the output is an integer tensor, then the start and delta must be integers too. It would be legal to pass start = -5 though. Otherwise DirectML will complain (fail API validation) that the `DML_FILL_VALUE_SEQUENCE_OPERATOR_DESC::ValueDataType` does not match the `outputTensorDesc->Type`.

Reply:
So we need use integer numbers for such cases,  though Spec define **start** and **step** as **float**, or does it need update this Spec?

```
start: a [float](https://webidl.spec.whatwg.org/#idl-float) scalar. The starting value of the range.

step: a [float](https://webidl.spec.whatwg.org/#idl-float) scalar. The gap value between two data points in the range.
```

Reply:
Yes, we should update the spec with additional wording. Even if you pass it as a float/double, internally it will be casted to an integer before any calculations are done with it. So then the baseline should cast any floating-point values (start and delta values) to integers (or BigInt) before computing.

Reply:
For now, for the integer cases, can we just pass integer start and delta values?
` "data": -5.520709991455078`, -> ` "data": -5,`

Reply:
Sure, I'll update test data for those integer cases.

Reply:
Done, please have another look, thanks.

Reply:
One case with a negative step for the integers would be good, but LGTM.

Reply:
Added. Please take another look, thanks.